week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -15,7 +25,17 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+const isNumber = typeof(num);
+const isEven = num % 2;
+
+if( isNumber === "number" && isEven === 0 && num <=100 ){
+return true
+}
+else{
+  return false
+}
```

Comment:
It's a correct solution but do we really need the if-else statement?
Think about it, what is the outcome type of that logical expression there? In other words, what type of value will it result?

Reply:
I am sorry, your point is not very clear for me

Reply:
I'm trying to give a more specific hint:

So your logical expression return a Boolean value? It can be either `true` or `false`. 
`val isNumericalValue = isNumber === "number" && isEven === 0 && num <=100`

Putting it into your example:
```
if(isNumericalValue){
  return true
}
else{
  return false
}
```
So if it is true then code proceeds with the `if` block, otherwise`, `else` block.

Do we really need to specify manually `return true` or `return false? 

What if you write this instead?

```
val isNumericalValue = isNumber === "number" && isEven === 0 && num <=100`;
return isNumericalValue;

Or you can make it even more compact:
return isNumber === "number" && isEven === 0 && num <=100
```